src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2603,20 +2625,40 @@ def _ensure_default_log_analytics_workspace_for_monitoring(cmd, subscription_id,
 
     # sanity check that locations and clouds match.
     if ((cloud_name.lower() == 'azurecloud' and AzureChinaRegionToOmsRegionMap.get(rg_location, False)) or
-            (cloud_name.lower() == 'azurecloud' and AzureFairfaxRegionToOmsRegionMap.get(rg_location, False))):
+            (cloud_name.lower() == 'azurecloud' and AzureFairfaxRegionToOmsRegionMap.get(rg_location, False)) or
+            (cloud_name.lower() == 'azurecloud' and AzureUSNatRegionToOmsRegionMap.get(rg_location, False)) or
+            (cloud_name.lower() == 'azurecloud' and AzureUSSecRegionToOmsRegionMap.get(rg_location, False))):
```

Comment:
Will the following code work as well?
`if (cloud_name.lower() == 'azurecloud' and not AzureCloudRegionToOmsRegionMap.get(rg_location, False):`

Reply:
code removed